### ISSUE TITLE
skip links

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/o-header#136]
Next has
```
<a class="n-util-visually-hidden" href="#primary-nav">Skip to navigation</a>
<a class="n-util-visually-hidden" href="#site-content">Skip to content</a>
```
which is good accessibility practice, and should be in the origami component

**Comment by AlbertoElias** [comment]
I'm not sure I understand what you mean

**Comment by wheresrhys** [comment]
https://www.google.co.uk/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=skip%20links

**Comment by AlbertoElias** [comment]
Oh, right. I think this should be completely up to products and shouldn't be integrated in the module

**Comment by wheresrhys** [comment]
On what grounds? It's accessibility best practice and origami should have a good reason for not including it in the header.

**Comment by AlbertoElias** [comment]
Because it doesn't seem to be something that should go inside the `<header>` elemen but above or below it. I hadn't heard about this accessibility idea, so I'm just happy to do whatever the experts say :)

**Comment by AlbertoElias** [comment]
Pinging @alicebartlett

**Comment by alicebartlett** [comment]
All pages should have skip links. They should be optional, but there by default, as I can see a case where a site might implement o-header but have it's own wrapper that has a skip link already.

@AlbertoElias a good way to check out how other sites handle the ux of skip links is to go to say, bbc.co.uk, or gov.uk and press the tab key.

**Comment by GlynnPhillips** [comment]
Another improvement might be to have these skip links become visible on `focus` which currently doesn't happen on Next.

**Comment by onishiweb** [comment]
Going to be discussed today in the open meeting
